src/test/projects/skip-cache-param/src/main/java/Main.java

```diff
@@ -0,0 +1,6 @@
+public class Main
+{
```

Comment:
```suggestion
public class Main {
```
code formatting: `{` should be on the same line